src/app/dashboard/payments/utils/PaymentList.tsx

```
@@ -0,0 +1,473 @@
+        return visibleColumns === 'all' ? columns : columns.filter((col) => visibleColumns.has(col.key));
+    }, [visibleColumns]);
+
+    const renderCell = useCallback((data: PaymentResponseInterface, columnKey: React.Key) => {
+        const cellValue = data[columnKey as keyof PaymentResponseInterface] as string | number | undefined;
+
+        switch (columnKey) {
+            case 'name':
+            case 'role':
+            case 'currency':
+            case 'payer_currency':
+                const Icon = cryptoIcons[(cellValue as string).toLowerCase() as keyof typeof cryptoIcons];
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)